.github/workflows/test-all-packages.yml

```diff
@@ -7,6 +7,12 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
+env:
+  AGORIC_AVA_USE_TAP: true
+  DD_ENV: ci
+  DATADOG_SITE: us3.datadoghq.com
+  DATADOG_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+  TEST_COLLECT: 'tee -a _testoutput.txt'
```

Comment:
good idea.

I still don't know whether you tried [this](https://github.com/Agoric/agoric-sdk/pull/6855#discussion_r1088311057) for `yarn test (everything)` but this addresses my noise concern enough.